composer.json

```diff
@@ -43,7 +43,7 @@
         "symfony/symfony": "3.4.*",
         "twbs/bootstrap": "~3.3.0",
         "twig/extensions": "~1.5",
-        "twig/twig": "~1.38"
+        "twig/twig": "1.41.0"
```

Comment:
Why is this hardcoded?

Reply:
See: https://github.com/OpenConext/Stepup-RA/pull/229/commits/8b7b8f430d1a5eb91cac677674e69e46ed41a4db